### ISSUE TITLE
update gif on readme.md with latest notebook UX

### ISSUE THREAD

**Issue by bcolloran** [iodide-project/iodide#1285]

**Comment by bcolloran** [comment]
this can be done once #1390 is merged

**Comment by bcolloran** [comment]
@hamilton with #1390 in place, mind cutting us a fancy new whirly gif of the beautiful purpley notebook :-)

**Comment by hamilton** [comment]
yep

**Comment by bcolloran** [comment]
looks good to me. the gif shows some sluggishness that we'll need to fix, but that's out of scope for this task. thanks!
